test/tools/unit_test_runner.d

```diff
@@ -234,6 +234,7 @@ void writeCmdfile(string path, string runnerPath, string outputPath,
         "-version=MARS",
         "-unittest",
         "-J" ~ buildOutputPath,
+        "-J" ~ projectRootDir,
```

Comment:
This is wrong. `VERSION` should be fetched from the generated folder. The static `VERSION` from the main directory should not be used. The Posix Makefiles and `build.d` generate the `VERSION` into generated as part of building dmd.

Reply:
Hmm, VERSION is a plain copy under Win32 so far and used directly for the VS projects. Would it be ok to version the option for Windows?

Reply:
If you must, though (if possible) it would be better to change the windows builds scripts as other places also depend on `VERSION` being put in `generated`.

Reply:
now changed the VS project to copy VERSION file to the generated folder just as win32.mak does.